app/models/submission.rb

```diff
@@ -93,6 +93,14 @@ def sheet_errors
     Hash[sheet_names.map { |sheet_name| [sheet_name, errors_for(sheet_name)] }]
   end
 
+  def file_key
+    files.first&.file_key
```

Comment:
When can there be more than one file? What does this represent when there is? Is there an implicit ordering that means that the first file is most recent?

Reply:
there is currently only ever one file. There was some thoughts back before I was involved that there might be more than one file, so it was made as a has_many, but this never came true.